jac-uk/apply
# #907 XSS vulnerability when using v-html to render HTML

## What's included?
Use [vue-dompurify-html](https://github.com/LeSuisse/vue-dompurify-html/tree/vue-legacy) to sanitise HTML when using `v-html`.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Preview URL: https://jac-apply-develop--pr908-feature-907-xss-vuln-qrbhylh3.web.app

1. Go to admin site and enter some test scripts (e.g. `<script>alert('XSS')</script>`, `<svg onload="alert(1)" ></svg>`) in `Role summary` or `About the role` field of an exercise.
2. Go to vacancies page on apply site and check if the scripts execute. 

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

---
PREVIEW:DEVELOP
